set.mm

```diff
@@ -653832,6 +653837,25 @@ D Fn ( I ... ( M - 1 ) ) /\
     $( $j usage 'sn-dtru' avoids 'ax-8'; $)
   $}
 
+  ${
+    $d ph t $.  $d ps t $.  $d x t $.
+    $( Class abstractions in a subclass relationship, closed form.  One
+       direction of ~ ss2ab using fewer axioms.  (Contributed by SN,
```

Comment:
Can we add a `$j usage` to provide a bit more specificity to "fewer axioms"?

Reply:
Something about the github mechanics led to this comment being called "outdated" so I made another comment. But both of them are about `ss2ab1` so you only need to pay attention to them once.